tests/foreman/api/test_repository_set.py

```diff
@@ -1,4 +1,4 @@
-"""Unit tests for the ``repository_sets`` paths.
+"""Tests for `katello/api/v2/products/<product_id>/repository_sets``.
 
 A full API reference for products can be found here:
```

Comment:
```suggestion
API reference for sync plans can be found on your Satellite:
```
because those katello.org URLs no longer work